src/GethNativeModule.ts

```diff
@@ -0,0 +1,24 @@
+import {Account, NodeConfig} from "./types";
+
+/**
+ * GethNativeModule defines the interface for the native modules:
+ * iOS:     RNGeth.m (exposed from RNGeth.swift)
+ * Android: RNGethModule.java (marked with @ReactMethod)
+ *
+ * We currently have more methods implemented in Android then in iOS,
+ * but this interface should hold the lowest common denominator.
+ * We should extend this list as we implement/start using more.
+ */
+export interface GethNativeModule {
```

Comment:
I appreciate moving to typescript and out of flow
do we really need another interface for this though?

Reply:
Not sure what you mean. This is the interface for the native module. This can't be autogenerated afaik. This is then consumed in `src/RNGeth.ts`:

```
geth: GethNativeModule = NativeModules.RNGeth
```

So the `RNGeth` class can have typing information about what it can call on the native module. And it also serves another purpose which is to have a clear place where we signal what native methods are available to call because in the old version of this there were methods implemented on the javascript side for various native methods that weren't available on iOS, just Android. But that wasn't clear unless you dove deep into the native code.

Reply:
This seems worthwhile if RNGeth <-> GethNativeModule aren't 1:1 mapped for every method, but it seems like they are. Will that change?